index.html

```diff
@@ -140,11 +115,15 @@ <h2 data-key="repo-id"><a href="https://www.github.com/heremaps/{repo-id}" targe
 			$.getJSON('https://api.github.com/orgs/heremaps/repos')
 			.done(function (repo_data) {
 				repo_data.forEach(function (data, index) {
+					/* The GitHub API will return more repos than the ones we expose here.
+						 Target only existing repo sections.*/
 					var $target = $('#repo-info-' + data.name);
 					if ($target.length > 0) {
-						$target.find('[data-role="repo-lastUpdate"]').text(moment(data.updated_at).format('ll'));
-						$target.find('[data-role="fork-count"]').text(data.forks_count);
-						$target.find('[data-role="star-count"]').text(data.stargazers_count);
+						$target.find('[data-key="repo-type"]').text(data.language);
```

Comment:
BTW, I found https://github.com/github/linguist/issues/3229 about the submodules issue. Still worth trying the path override. But thinking about it more, it seems odd to use `.gitattributes` in one repo (the superproject) to describe the language in another repo (the submodules). We should probably keep things together, and implement the override in the JavaScript code.